notebooks/dc2/catalog_generation/matchingLensGalaxies.ipynb

```diff
@@ -5146,13 +5198,21 @@
     "        shear_2_lenscat = gamma_lenscat * np.sin(2 * phi_gamma_lenscat)\n",
     "        sindex_lens = 4\n",
     "\n",
-    "        major_axis_lens = matched_sys_cat.iloc[i]['reff_lens'] / \\\n",
-    "                            np.sqrt(1 - matched_sys_cat.iloc[i]['ellip_lens'])\n",
-    "        minor_axis_lens = matched_sys_cat.iloc[i]['reff_lens'] * \\\n",
-    "                            np.sqrt(1 - matched_sys_cat.iloc[i]['ellip_lens'])\n",
-    "        position_angle = matched_sys_cat.iloc[i]['phie_lens']*(-1.0)*np.pi/180.0\n",
-    "        ellip_lens = matched_sys_cat.iloc[i]['ellip_lens']\n",
-    "        phie_lens = matched_sys_cat.iloc[i]['phie_lens']\n",
+    "#         major_axis_lens = matched_sys_cat.iloc[i]['reff_lens'] / \\\n",
+    "#                             np.sqrt(1 - matched_sys_cat.iloc[i]['ellip_lens'])\n",
+    "#         minor_axis_lens = matched_sys_cat.iloc[i]['reff_lens'] * \\\n",
+    "#                             np.sqrt(1 - matched_sys_cat.iloc[i]['ellip_lens'])\n",
+    "#         position_angle = matched_sys_cat.iloc[i]['phie_lens']*(-1.0)*np.pi/180.0\n",
+    "        major_axis_lens = matched_lenses.iloc[i]['size']\n",
+    "        minor_axis_lens = matched_lenses.iloc[i]['size_minor']\n",
+    "        position_angle = matched_lenses.iloc[i]['position_angle']\n",
+    "\n",
+    "        # Change dc2 ellip [(1-q)/(1+q)] to om10 ellipticity which is (1-q)\n",
+    "        q = minor_axis_lens/major_axis_lens\n",
+    "        ellip_lens = matched_lenses.iloc[i]['ellipticity'] * (1+q)\n",
```

Comment:
You can also just do `ellip_lens = 1.0 - q`

Reply:
Haha, oh yeah, *facepalm*.